docs/user/whats-new.asciidoc

```diff
@@ -5,12 +5,254 @@ Here are the highlights of what's new and improved in {minor-version}.
+=== ResponseOps
+
+[discrete]
+==== New {esql} rule type
+
+A new {esql} alerting rule type is now available under the existing {es} rule type. This rule type brings all the new functionalities that are available within the new and powerful language, {esql}, to {kib} Alerting to allow and unlock new alerting use cases.
+
+With the new type, users will be able to generate a single alert based on defined {esql} query and preview the query result before saving the rule. When the query returns an empty result no alerts will be generated.
+
+[role="screenshot"]
+image::images/esql-rule-type.gif[An example of creating a rule type using {esql}]
+
+[discrete]
+==== {kib} cases custom field
+
+A new functionality is now available in Kibana cases. Users will be able to add custom fields to the case structure so they can use it for better classification and case enrichment. As a first step those fields will be available in the case view only, but next step we’re planning to have more fields types, dedicated privilege, support filters, and search capabilities in the case table accordingly.
+
+[role="screenshot"]
+image::images/custom-field.gif[An example of creating a custom case field in {kib}]
+
+[discrete]
+==== Supporting multi levels of term aggregations in {es} rule type 
+
+The existing {es} alerting rule (KQL-based) is now supported by multiple selection when grouping by alert fields which allows you to define multiple layers of term aggregations.
+
+[role="screenshot"]
+image::images/term-aggs.png[An example of creating multiple layers of term aggregations]
+
+[discrete]
+==== Slack connector - allow List
+
+The Slack connector supports a new allow list so customers will be able to manage the available slack channels within the alert actions. 
+
+[discrete]
+=== {observability}
+
+[discrete]
+==== {es} alerting rule availability in {observability} solution
+
+The existing {es} alerting rule is now available in {observability}. Before this update, {observability} customers who leverage the {es}  rule were required to move between {stack-manage-app} and {observability} to manage their alerts.
+To manage the {observability} roles properly, we added a new field to determine which role can maintain the created rule and its generated alerts. 
+
+[role="screenshot"]
+image::images/alerting-rule.png[An example of creating an altering rule in {observability}]
+
+[discrete]
+=== Global Experience 
+
+[discrete]
+==== Create and manage {esql} enrichment index policies
+
+In support of our new {esql} capabilities, we have added link:{ref}/ingest-enriching-data.html[enrich policies] to our **Index Management** experience. Users can now create their enrichment policies right from here and get started using it right away. Once configured, all enrich policies are available in the **Enrich Policies** tab. Existing policies can be managed right from the enrichment policies tab in index management.
+
+.Example of an ES|QL Query using an enrich policy with enrich:
```

Comment:
We don't use Figure titles.  Remove figure titles here and Figure titles for 1 & 2 earlier in the doc.